stable/jenkins/README.md

```diff
@@ -25,6 +25,80 @@ $ helm install my-release stable/jenkins
 
 A major chart version change (like v0.40.0 -> v1.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### 2.0.0 Configuration as Code now default + container does not run as root anymore
+
+#### Configuration as Code new default
+
+Configuration is done via [Jenkins Configuration as Code Plugin](https://github.com/jenkinsci/configuration-as-code-plugin) by default.
+That means that changes in values which result in a configuration change are always applied.
+In contrast the XML configuration was only applied during the first start and never altered.
+
+:exclamation::exclamation::exclamation: Attention: This also means if you manually altered configuration then this will most likely be reset to what was configured by default. :exclamation::exclamation::exclamation:
+
+`securityRealm` and `authorizationStrategy` are not configured using configuration as code so these should be identical to what you configured before.
+It's advised to supply your own configuration here. e.g.
+
+```yaml
+master:
+  JCasC:
+    configScripts:
+      security: |
+        jenkins:
+          authorizationStrategy:
+            loggedInUsersCanDoAnything:
+              allowAnonymousRead: false
+```
```

Comment:
I'm thinking this block can be removed. #22408 configures `securityRealm` and `authorizationStrategy` using configuration as code.

Reply:
I updated that section.